server/models/Card.js

```diff
@@ -0,0 +1,48 @@
+const mongoose = require("mongoose");
+
+const cardSchema = new mongoose.Schema(
+  {
+    name: {
+      type: Array,
+      required: false,
+    },
```

Comment:
Shouldn't `name` be required? Do users ever have the option to not put in a `name`?

Reply:
Make sure to change the type to `String` and yes, make it required